src/loader.js

```diff
@@ -0,0 +1,38 @@
+let isLoadingLibrary = false
+const callbacks = []
+
+function hasLoadedLibrary() {
+  return window.google && window.google.maps && window.google.maps.places
+}
+
+export function loadGooglePlacesLibrary(apiKey, callback) {
+  if (hasLoadedLibrary()) {
+    callback()
+    return
+  }
+  
+  callback && callbacks.push(callback)
+  
+  if (isLoadingLibrary) {
```

Comment:
a "why" comment would be helpful here.

Reply:
Addressed in the documentation. Would it be helpful to have a comment here instead / as well?

Reply:
should be ok up above.